generator/lib/src/generator.dart

```diff
@@ -1136,6 +1137,8 @@ class RetrofitGenerator extends GeneratorForAnnotation<retrofit.RestApi> {
                   ${p.displayName}.map((i) => MapEntry(
                 '${fieldName}',
                 MultipartFile.fromBytes(i,
+
```

Comment:
Please fix the whitespace + formatting of this line

Reply:
I've removed the whitespace and fixed the formatting, now it is like in the other occurrences of MultipartFile